recipes-core/images/includes/nilrt-image-initramfs.inc

```diff
@@ -0,0 +1,10 @@
+# Rules for image recipes which: contain a valid kernel and init (PID 1); are
+# packed as a CPIO archive; and expected to provide an initial ramfs for
+# booting a later, complete rootfs.
+
+IMAGE_FSTYPES += " ${INITRAMFS_FSTYPES}"
```

Comment:
Just a thought. To me, it seems like there isn't enough here to justify an includes file.

Reply:
You might be right; especially after we remove the runmode initramfs. I'll consider removing it.